src/iop/clipping.c

```diff
@@ -1187,12 +1187,6 @@ void commit_params(struct dt_iop_module_t *self, dt_iop_params_t *p1, dt_dev_pix
     d->all_off = 1;
     d->k_apply = 0;
   }
-  else
```

Comment:
Are you sure about this one?

We have:

```
if(p->k_type == 4)
else if(p->k_type >= 0 && p->k_apply == 1)
else if(p->k_type == 0)
else
```

So the else part can be taken if p->k_type > 0 and p->k_apply == 0.

Reply:
Out of own experience I'd be very cautious with any changes to that module shortly before a release. IMHO it's better to allow for intensive testing in the development branch.